pyrte_rrtmgp/rrtmgp_gas_optics.py

```diff
@@ -74,6 +74,9 @@ def load_gas_optics(
     return dataset
 
 
+REQUIRED_GASES: tuple = ("h2o", "co2", "o3", "n2o", "co", "o2")
```

Comment:
As a question of curiosity why a tuple and not a list here?

Reply:
@RobertPincus I choose a tuple because i thought the collection was immutable, but from the code context, a `set` also seems appropriate...or frozen set.  But just a starting point...and sounds like @sehnem may have a good idea here